### PR TITLE
Feature/search request adaptor

### PR DESCRIPTION
**Pull request by peggy-gov-bc-ca**
# Description

This PR includes the following proposed change(s):

- FAMS3-2411(SearchRequestAdaptor-create project and make rabbit mq.)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Tested locally
## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/SearchApi/SearchRequestAdaptor/Consumer/SearchRequestEventConsumer.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+        public SearchRequestOrderedConsumer(ILogger<SearchRequestOrderedConsumer> logger)
+        {
+            _logger = logger;
+
+        }
+
+        public async Task Consume(ConsumeContext<SearchRequestOrdered> context)
+        {
+            _logger.LogInformation("get the searchRequestOrdered message.");
```
Comment (rantifamilusi) [inline app/SearchApi/SearchRequestAdaptor/Consumer/SearchRequestEventConsumer.cs:20]: Can we remove this warning.

File: app/SearchApi/SearchRequestAdaptor/Notifier/SearchRequestNotifier.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        public WebHookNotifierSearchRequest(HttpClient httpClient, ILogger<WebHookNotifierSearchRequest> logger)
+        {
+            _httpClient = httpClient;
+            _logger = logger;
+        }
+
+        public async Task NotifyEventAsync(string searchRequestKey, SearchRequestEvent searchRequestEvent, string eventName,
+           CancellationToken cancellationToken)
+        {
+
```
Comment (rantifamilusi) [inline app/SearchApi/SearchRequestAdaptor/Notifier/SearchRequestNotifier.cs:33]: Can we remove this warning

File: app/SearchApi/SearchRequestAdaptor/Consumer/SearchRequestEventConsumer.cs (hunk 1 above)
Comment (peggy-gov-bc-ca) [reply app/SearchApi/SearchRequestAdaptor/Consumer/SearchRequestEventConsumer.cs:20]: sure

File: app/SearchApi/SearchRequestAdaptor/Notifier/SearchRequestNotifier.cs (hunk 2 above)
Comment (peggy-gov-bc-ca) [reply app/SearchApi/SearchRequestAdaptor/Notifier/SearchRequestNotifier.cs:33]: sure

**Pull request merged**